node/src/accountant/mod.rs

```diff
@@ -1599,6 +2036,148 @@ pub mod tests {
         );
     }
 
+    #[test]
+    fn accountant_receives_new_payments_to_the_receivables_dao() {
+        let wallet = make_wallet("wallet0");
+        let earning_wallet = make_wallet("earner3000");
+        let gwei_amount = 42u64;
+        let expected_payment = Transaction {
+            block_number: 7u64,
+            from: wallet.clone(),
+            gwei_amount,
+        };
+        let more_money_received_params_arc = Arc::new(Mutex::new(vec![]));
+        let receivable_dao = ReceivableDaoMock::new()
+            .more_money_received_parameters(&more_money_received_params_arc)
+            .more_money_received_result(Ok(()))
+            .more_money_received_result(Ok(()));
+        let accountant = make_accountant(
+            Some(bc_from_ac_plus_earning_wallet(
+                AccountantConfig {
+                    payables_scan_interval: Duration::from_secs(10_000),
+                    receivables_scan_interval: Duration::from_secs(10_000),
+                    pending_payments_scan_interval: Duration::from_secs(100),
+                    when_pending_too_long_sec: DEFAULT_PENDING_TOO_LONG_SEC,
+                },
+                earning_wallet.clone(),
+            )),
+            Some(PayableDaoMock::new().non_pending_payables_result(vec![])),
+            Some(receivable_dao),
+            None,
+            None,
+            None,
+        );
+
+        let system = System::new("accountant_receives_new_payments_to_the_receivables_dao");
+        let subject = accountant.start();
+
+        subject
+            .try_send(ReceivedPayments {
+                payments: vec![expected_payment.clone(), expected_payment.clone()],
+            })
+            .expect("unexpected actix error");
+        System::current().stop();
+        system.run();
+        let more_money_received_params = more_money_received_params_arc.lock().unwrap();
+        assert_eq!(1, more_money_received_params.len());
+
+        let more_money_received_params = more_money_received_params.get(0).unwrap();
```

Comment:
This more_money_received_params has a different type than the more_money_received_params it's replacing, which makes reading the code difficult.